cmd/oras/root/blob/delete.go

```diff
@@ -69,6 +71,8 @@ Example - Delete a blob and print its descriptor:
 		},
 	}
 
+	cmd.Flags().BoolVarP(&opts.verbose, "verbose", "v", true, "verbose output")
+	_ = cmd.Flags().MarkDeprecated("verbose", "and will be removed in a future release.")
```

Comment:
Can we refactor it out? Since the verbose variable completely has no effect, can we just point it to nowhere?

We can have a quick discuss offline for the trick.

Reply:
refactored after offline discussion